src/Servers/Kestrel/Core/src/Internal/ConfigurationReader.cs

```diff
@@ -346,6 +346,8 @@ internal class SniConfig
 
     // "CertificateName": {
     //     "Path": "testCert.pfx",
+    //     "KeyPath": "",
+    //     "ChainPath": "",
```

Comment:
Nit: Use fake `.pem` files to show the expected file type for KeyPath/ChainPath.